src/main/java/seedu/address/ui/StudentCard.java

```diff
@@ -53,7 +53,7 @@ public StudentCard(Student student, int displayedIndex) {
         email.setText(student.getEmail().value);
         remark.setText(student.getRemark().value);
         student.getTags().stream()
-                .sorted(Comparator.comparing(tag -> tag.tagName))
-                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+                .sorted(Comparator.comparing(tag -> tag.ranking))
+                .forEach(tag -> tags.getChildren().add(new Label(tag.ranking.toString())));
```

Comment:
could I know why the .forEach() method was used? will there be more than one tag assigned to a Student at any point in time?